.github/workflows/deploy.yaml

```diff
@@ -52,5 +52,7 @@ jobs:
           path: git-subrepo
       - run: |
           source ../git-subrepo/.rc
+          git config user.name github-actions
+          git config user.email github-actions@github.com
```

Comment:
```suggestion
          git config user.email action@github.com
```
I believe is the email for @actions-user

Reply:
I just was following https://github.com/actions/checkout#push-a-commit-using-the-built-in-token

Reply:
maybe that's out of date?  I wouldn't be shocked 😅

Reply:
:shrug: